### DISCUSSION TITLE
Bilder übereinander legen

### DISCUSSION THREAD

**Discussion by GrauerWolf-55** [tannerhelland/PhotoDemon#437] [General]
Ein nettes Moin

Wie kann man 2 oder 3 Bilder übereinander legen

MfG GrauerWolf-55

**Comment by hi5** [comment]
> How can you put 2 or 3 pictures on top of each other

One way:

1.  Open the first image
2.  Now drag and drop from explorer (or your file manager) the second image into the image in photodemon, it will ask "Open as new image" or "add it to current image as new layer" - Choose the layer option and you now have two images "on top of each other" - you can play around with the order and properties (opacity, blend, alpha)
3.  repeat step 2
